web/src/pages/Trace/useOpenGuidedTourForNewUsersEffect.tsx

```diff
@@ -1,14 +1,14 @@
 import {Dispatch, SetStateAction, useEffect} from 'react';
 
 const key = 'tracetest-new-user';
-const value = String(true);
+const oldUserValue = String(true);
 
 export function useOpenGuidedTourForNewUsersEffect(setVisible: Dispatch<SetStateAction<boolean>>): void {
```

Comment:
The `web/src/services/GuidedTour.service.ts` already handles this.

Reply:
There are some commented lines here `web/src/hooks/useGuidedTour.ts` that I think are doing the same thing